source/user-manual/agents/remote-upgrading/upgrading-agent.rst

```diff
@@ -194,11 +202,11 @@ Using the RESTful API
             "affected_items": [
               {
                 "id": "002",
-                "version": "Wazuh v4.2.2"
+                "version": "Wazuh v4.0.0"
```

Comment:
You used the endpoint without any `version` parameter so they should upgrade to the latest version. I do not understand this change.

Reply:
I wanted to add the `version` parameter in `PUT /agents/upgrade` so this change would make sense. 

I just forgot it, doing it in a new commit.